### PR TITLE
more cleanup

### PR DESCRIPTION
**Pull request by xmunoz**
- Rename json files with example appended
- Add contact info in developers
- More pylint cleanup:
  - lazy string interpolation for logging statements
  - remove unused imports
  - remove useless method

### CODE REVIEW

File: engine/ai.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ class AI:
     def __init__(self):
         self.owner = None  # Fully mutable member.
 
-    def choose_action(self, room, impulse=None):
+    def choose_action(self, unused_room, impulse=None):
```
Comment (flosincapite) [inline engine/ai.py:45]: Why not "unused_impulse," too?

File: engine/when.py [hunk 2]
```diff
@@ -23,8 +21,8 @@ def _add_parameter_strings(parameter, kind_to_argstrings, kind_to_callstrings):
         arg_string = f"**{parameter.name}"
         call_string = f"**{parameter.name}"
     if parameter.default is not inspect.Parameter.empty:
-        arg_string = f"{parameter_string}={parameter.default}"
-        call_string = f"{parameter_string}={parameter_string}"
+        arg_string = f"parameter_string={parameter.default}"
```
Comment (flosincapite) [inline engine/when.py:24]: Oh, I see. This has been fucked up 'n otiose for a while. The correct fix here is

```
        arg_string = f"{parameter.name}={parameter.default}"
        call_string = f"{parameter.name}={parameter.name}"
```

**Review by flosincapite — COMMENTED** [review]
If you move `logging_config.json` and `game_config.json`, it will break things, as code in `__main__.py` and `game.py` expects those files to be there. If you review the state of the code before #104, you can see how the default values were obtained.

File: engine/when.py (hunk 2 above)
Comment (flosincapite) [inline engine/when.py]:
```suggestion
        arg_string = f"{parameter.name}={parameter.default}"
```

File: engine/when.py [hunk 3]
```diff
@@ -23,8 +21,8 @@ def _add_parameter_strings(parameter, kind_to_argstrings, kind_to_callstrings):
...
+        call_string = "parameter_string=parameter_string"
```
Comment (flosincapite) [inline engine/when.py]:
```suggestion
        call_string = f"{parameter.name}={parameter.name}"
```

File: engine/ai.py (hunk 1 above)
Comment (xmunoz) [reply engine/ai.py:45]: Because I'm just doing this so that the method signatures match :)

**Comment by xmunoz** [comment]
> If you move `logging_config.json` and `game_config.json`, it will break things, as code in `__main__.py` and `game.py` expects those files to be there. If you review the state of the code before #104, you can see how the default values were obtained.

Not sure what to do about this. `.gitignore` doesn't work the way that we thought it would. If you add something to the index, it can't be ignored.

**Comment by flosincapite** [comment]
> > If you move `logging_config.json` and `game_config.json`, it will break things, as code in `__main__.py` and `game.py` expects those files to be there. If you review the state of the code before #104, you can see how the default values were obtained.
> 
> Not sure what to do about this. `.gitignore` doesn't work the way that we thought it would. If you add something to the index, it can't be ignored.

Yeah, I think it was a bad idea to mess with them in the first place. So let's do this. We keep `logging_config.json` and `game_config.json` in `.gitignore` but read from something like `logging_config.default.json`. That way, we can still change the default settings as needed. In code, we'll override the defaults with custom settings. It will be identical to the situation we had before your suggestion, except now we'll be reading the default values from code.

**Review by flosincapite — APPROVED** [review]

**Pull request merged**